index.bs

```diff
@@ -203,11 +203,7 @@ enum OperandType {
   "float32",
   "float16",
   "int32",
-  "uint32",
-  "tensor-float32",
-  "tensor-float16",
-  "tensor-int32",
-  "tensor-quant8-asymm"
+  "uint32"
```

Comment:
Do we need to support `int8` and `uint8`?

Reply:
Agreed. Added.